mkdocs.yml

```diff
@@ -6,6 +6,9 @@ pages:
   - Home: 'index.md'
   - Configuration Reference: 'configuration-reference.md'
   - REST API: 'rest-api.md'
+  - Tutorial: 'tutorial/introduction.md'
```

Comment:
I was wondering if the Contributing part should be higher in the menu. What do you think?

Reply:
I would leave it towards the bottom. Think of it in order of how many people will be interested in a page. I think there will be fewer people wanting to contribute than anything else really.